web/src/routes/stall/[...slug].svelte

```diff
@@ -0,0 +1,59 @@
+<svelte:head>
+    <title>Market Stall</title>
+</svelte:head>
+
+<script context="module">
+    export async function load({ params }) {
+        const { slug } = params;
+        return { props: { stallName: slug } }
+    }
+</script>
+
+<script lang="ts">
+    import { token, user, Info } from "$lib/stores";
+    import { Auction, fromJson } from "$lib/types/auction";
+    import StallView from "$lib/components/StallView.svelte";
+    import ListView from "$lib/components/ListView.svelte";
+    import AuctionCard from "$lib/components/AuctionCard.svelte";
+    import AuctionEditor from "$lib/components/AuctionEditor.svelte";
+    import { getProfile } from "$lib/services/api";
+    import { onMount } from "svelte";
+    import { afterNavigate } from '$app/navigation';
+
+    export let stallName: string;
```

Comment:
We should not call this stallName, since it is not really the name. We might add a name/description in the future and get really confused. It is really the stallOwnerNym, or *maybe* a  stallSlug. But definitely not stallName!